code-studio/src/js/clientState.js

```diff
@@ -251,9 +376,9 @@ function recordVisualElementSeen(visualElementType, visualElementId) {
  * @param visualElementId
  */
 function hasSeenVisualElement(visualElementType, visualElementId) {
-  var elementSeenJson = sessionStorage.getItem(visualElementType) || '{}';
+  var elementSeenJson = getItem(visualElementType) || {};
   try {
-    var elementSeen = JSON.parse(elementSeenJson);
+    var elementSeen = elementSeenJson;
     return elementSeen[visualElementId] === true;
```

Comment:
Done.

Lscache catches parsing exceptions and returns the raw string if it is unable to parse the local storage item as JSON.  

To handle this case, I added some code to fall back on the empty value if the return result from getitem is not an object in those cases when we are expecting one.  This should never happen unless these are bugs or data corruption.